content/ui-patterns/danger.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Danger patterns
+---
+
+import Img1 from '../../images/danger/danger1.png'
+import Img2 from '../../images/danger/danger2.png'
+import Img3 from '../../images/danger/danger3.png'
+import Img4 from '../../images/danger/danger4.png'
+import Img5 from '../../images/danger/danger5.png'
+import Img6 from '../../images/danger/danger6.png'
+import Img7 from '../../images/danger/danger7.png'
+
+## Destructive changes
+
+When users are making a destructive change to their GitHub account, you should make sure they have a clear understanding of what is about to happen. Many times these changes are reversible, but may have irreversible side effects (making a repository public). Other times, they're not reversible at all (deleting an organization). Either way, ensure that you've done everything you can so there are no gaps in communication that may cause unintended effects so that GitHub doesn't lose users' trust.
+
+### Non-critical data
+
+There are several types of data within GitHub that will need to be deleted from time to time. Many of these are non-critical and can be handled without using a `danger` visual treatment. These non-critical destructive flows include deleting a comment, issue, pull request, or discussion.
+
+### Critical data
+
+For critical destructive flows (including modifying user access, deleting repositories, making private data public, etc.), there are 3 basic treatments you can apply that will ensure the full effects of a change are clear to users and that we minimize loss of trust.
+
+#### Visual treatment
+
+1. Highlight all destructive flows visually. 
+    - All page sections that will trigger a destructive sequence should be highlighted with a [`red border`](https://primer.style/css/utilities/borders#border-colors) and an [`alert` octicon](https://primer.style/octicons/alert-16) to build an accessible, intuitive pattern of recognition. When users notice those details, they should know to tread carefully. Ideally, these will be centralized into a "Danger Zone" section.
+    - Any actions that will delete significant user data must use the [`danger button`](https://primer.style/css/components/buttons#danger-button) component.
+
+<img alt='Visual treatment of Danger Zone' src={Img1} />
```

Comment:
@ashygee @vdepizzol In these images where there is a number/caption, do we want to use another style to highlight the relevant part of the image? Or keep the numbers? [Figma](https://www.figma.com/file/OiKTijm0ROzZ7LI7sgBvTy/Danger-patterns?node-id=164%3A791)